lib/parsers/index.ts

```diff
@@ -14,7 +14,7 @@ export interface PkgTree {
 }
 
 export interface DotnetDepsPkgTree extends PkgTree {
-    targetFramework?: string;
+    targetFramework?: string | string[];
```

Comment:
I think I see what you meant now, how is this handled in `dotnet-dps` now that it is both single || array?  Before we merge this need to add a PR to dotnet-deps to expect both and process accordingly

Reply:
Also when thinking about it, it makes more sense to switch to `string[]` and have 1-element long array. Also the name would change to `targetFrameworks`. I know it's more work, but it will help us to understand, what's going on in the future.

Reply:
@lili2311 Well, we were both wrong :-) @RyanHavoc is right - this is a library, not a service, so as long as the version is pinned properly in `dotnet-deps`, nothing should happen. Our comments would be valid if this would be a service.